stable/aws-pod-identity-webhook/templates/mutatingwebhook.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
```

Comment:
Could support be added here to configure `annotations`? We've pulled this to use it, but want to use it with self-signed certificates from cert-manager instead.